src/pt_br/index.js

```diff
@@ -727,3 +815,37 @@ export const i19TransactionsType = {
   payment: 'Pagamento',
   recurrence: 'Assinatura'
 }
+
+export const i19TriggersAction = {
+  create: 'Criar',
+  change: 'Alterar',
+  delete: 'Deletar'
+}
+
+export const i19TriggersMethods = {
+  post: 'POST',
+  patch: 'PATCH',
+  put: 'PUT',
+  delete: 'DELETE'
+}
+
+export const i19TriggersResource = {
```

Comment:
```suggestion
export const i19StoreApiResources = {
```

Podia chamar isto de `i19StoreApiResources` e declarar todos os recursos da Store API (não só os disponíveis pra triggers)

Reply:
Então adicionaria `stores`, `procedures` e o próprio `triggers`